src/utils/helpers/helper.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { YT_API_KEY } from '../constants';
+
+export const youtubeSearchClient = axios.create({
+  baseURL: 'https://www.googleapis.com/youtube/v3/search',
+  params: {
+    part: 'snippet',
```

Comment:
This is good. However, you are not passing the query input from the search bar, so it does not matter what the user types, he is always receiving the same list of videos. You need to filter from the user search entry.